common/org.testeditor.dsl.common.ui/src/org/testeditor/dsl/common/ui/utils/ProjectContentGenerator.xtend

```diff
@@ -142,6 +146,25 @@ class ProjectContentGenerator {
 		]
 	}
 
+	def createGradleSettings(IProject project, IProgressMonitor monitor) {
+		var Properties props = System.getProperties();
+		if (!isNullOrEmpty(props.getProperty("http.proxyHost"))) {
+			var IFile gradleProperties = project.getFile("gradle.properties")
```

Comment:
The problem I see here is that the `gradle.properties` is usually checked in. So if we write the username and password for the proxy into that file we have a problem if those credentials are personalized.